userspace/ksud/src/ksu.rs

```diff
@@ -208,6 +210,13 @@ pub fn root_shell() -> Result<()> {
         }
     }
 
+    // add /data/adb/ksu/bin to PATH
+    #[cfg(any(target_os = "linux", target_os = "android"))]
+    {
+        add_path_to_env("/data/adb/ksu/bin")?;
+        link_ksud_to_bin();
```

Comment:
It is better to move this to https://github.com/tiann/KernelSU/blob/main/userspace/ksud/src/event.rs#L246